homeworks/valentin-vaninetti/01-html/contact.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+          name="viewport">
+    <meta content="ie=edge" http-equiv="X-UA-Compatible">
+    <title>Document</title>
+</head>
+<body>
+<header>
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="contact.html">Contact</a>
+    </nav>
+</header>
+<hr/>
+<article>
+    <h1> Contact Me</h1>
+    <form method="POST">
```

Comment:
for our mock example maybe its better to add `href=#` attribute in form tag to void the post behaviour

Reply:
Done!